includes/admin/views/html-report-by-date.php

```diff
@@ -8,9 +8,17 @@
 if ( ! defined( 'ABSPATH' ) ) {
 	exit; // Exit if accessed directly.
 }
-
 ?>
-
+<div id="message" class="error inline">
+	<p>
+		<strong>
+		<?php
+		/* translators: 1: Link URL */
+		echo wp_kses_post( sprintf( __( 'With the release of WooCommerce 4.0, these reports are being replaced. There is a new and better Analytics section available for users running WordPress 5.3+. Head on over to the <a href="%1$s">WooCommerce Analytics</a> or learn more about the new experience in the <a href="https://docs.woocommerce.com/document/woocommerce-analytics/" target="_blank">WooCommerce Analytics documentation</a>.', 'woocommerce' ), esc_url( wc_admin_url( '&path=/analytics/overview' ) ) ) );
+		?>
```

Comment:
I've submitted a rewording of the docs to the following new text:


> With the release of WooCommerce 4.0 in March 2019, these reports are deprecated and won’t receive further updates.
> There is a new and better Analytics section available for users running WordPress 5.3+. Learn more about the new reports and admin experience in the WooCommerce Analytics documentation.

So if it gets approved, we can also change it here.